docs/_deployments/serverless-framework.md

```diff
@@ -0,0 +1,342 @@
+---
+title: Serverless Framework
```

Comment:
In the side navation, I've chose to name the guide "Serverless Framework". The problem with this is that it doesn't jump out for someone looking for "AWS Lambda".

A few ideas:
1. Add a third side nav item called "AWS Lambda" that goes to this guide
1. Keep "Serverless Framework"
1. Rename to "AWS Lambda"

![image](https://user-images.githubusercontent.com/21328/100797167-8a39e500-33d6-11eb-8b21-bc61b8eb5cce.png)

Reply:
Having the below list sounds easier to understand to me because serverless framework is not a platform but a tool.

* Heroku Platfform
* AWS Lambda

As the main topic here is about AWS and we use serverless framework as a solution for simpler deployments, I think we can rename it to "AWS Lambda".

Reply:
Thanks for your suggestion @seratch. I agree with you and I like the reasoning that Serverless Framework is a solution for simpler deploys, but we're deploying to AWS Lambda.

Reply:
In commit c8c8f42 I've updated our guide to be:
- Title: "Deploying to AWS Lambda"
- Path: `/aws-lambda`
- Side navigation: "AWS Lambda"
- Sample app: "Getting Started with AWS Lambda"

In the next commit, I'll rename:
- Guide filename: `aws-lambda.md`
- Sample app: `/examples/getting-started-with-aws-lambda`